subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -216,6 +216,26 @@ static void net_key_list(struct bt_mesh_model *model,
 	k_sem_give(&cli->op_sync);
 }
 
+static void node_reset_status(struct bt_mesh_model *model,
+		struct bt_mesh_msg_ctx *ctx, struct net_buf_simple *buf)
+{
+	struct bool *param = NULL;
```

Comment:
`struct  bool *param = NULL;` and below `*param->status`

Reply:
`bool` is a standard type defined by C programs, and other type declarations are not allowed as prefixes.

Reply:
You have some build error, please click Details in `buildkite/zephyr` below for more information.